_data/learning.yml

```
@@ -93,7 +93,7 @@ reference-links:
     url: https://j3-fortran.org/
     description: J3 is the US National Body for the international Fortran standards committee
   
-  - name: "WG5: International Fortran Standards Committe"
+  - name: "WG5: International Fortran Standards committee"
```

Title case is used here
```suggestion
  - name: "WG5: International Fortran Standards Committee"
```